terraform/trial_rg/services.tf

```diff
@@ -52,6 +54,7 @@ module "trial_app_service_practitioner" {
 
   # todo use private endpoint
   settings = {
+    "MANAGED_IDENTITY"                        = module.trial_app_service_init.identity
```

Comment:
is this all it takes to allow this identity to access this service?

Reply:
or is it our own new env var, which will be used in the services code

Reply:
it will be available to the services code via a call similar to..
env.getProperty(Managed_Identity)